multi-storage-transaction-sample/docs/README.md

```diff
@@ -0,0 +1,260 @@
+# Multi-storage Transaction Sample
+
+This is a sample application for Multi-storage Transaction in ScalarDB.
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Sample application
+
+### Overview
+
+This is a simple EC application where you can order items and pay with a credit card using ScalarDB.
+
+In this article, you create the sample application on Cassandra and MySQL.
+With Multi-storage Transaction in ScalarDB, you can execute a transaction that spans Cassandra and MySQL.
+Please note that application-specific error handling, authentication processing, etc., are omitted in the sample application since it focuses on explaining how to use ScalarDB.
+Please see [this document](https://github.com/scalar-labs/scalardb/blob/master/docs/api-guide.md#handle-exceptions) for the details of how to handle exceptions in ScalarDB.
+
+![Overview](images/overview.png)
```

Comment:
Since this doc contains images, can we add the "multi-storage-transaction-sample/images" folder to the "multi-storage-transaction-sample/docs" folder as well?

Reply:
Good catch! I completely forgot it

Reply:
Done https://github.com/scalar-labs/scalardb-samples/pull/26/commits/e4f268f671db43bbe53f80a7926aca49b5129fab

```
$ for dir in $(find . -type d -d 1 -name '*-sample'); do cp -pr $dir/images $dir/docs/; done
cp: ./scalardb-graphql-sample/images: No such file or directory
cp: ./scalardb-server-sample/images: No such file or directory
$ for dir in $(find . -type d -d 1 -name '*-sample'); do diff -r $dir/images $dir/docs/images; done
diff: ./scalardb-graphql-sample/images: No such file or directory
diff: ./scalardb-graphql-sample/docs/images: No such file or directory
diff: ./scalardb-server-sample/images: No such file or directory
diff: ./scalardb-server-sample/docs/images: No such file or directory
```